src/main/java/com/mael/ttt/players/PlayerCreator.java

```diff
@@ -16,19 +15,26 @@ public PlayerCreator(UserInterface gameUI) {
     }
 
     public Player createPlayer(MenuOption option) {
-        return createPlayerOfType(option.getPlayerType(), PLAYER);
+        return createPlayerOfType(option);
     }
 
     public Player createOpponent(MenuOption option) {
-        return createPlayerOfType(option.getOpponentType(), OPPONENT);
+        return createOpponentOfType(option);
```

Comment:
What is the point of having `createOpponent` and `createOpponentOfType`? Their signature is identical except for the name. The same goes for `createPlayer` and `createPlayerOfType`.